app/javascript/lib/validation/validate-file-content.js

```diff
@@ -322,10 +349,11 @@ async function parseFile(file, fileType) {
         return { table, delimiter, issues: [['error', 'cap:format:no-newlines', 'File does not contain newlines to separate rows']] }
       }
       // if there are no encoding issues, and this isn't a gzipped file, validate content
-      delimiter = sniffDelimiter(lines, mimeType)
+      delimiter = sniffDelimiter(headerLines, mimeType)
       table = lines.map(line => line.split(delimiter))
-
-      issues = await validateCapFormat(table, fileType)
+      const headerTable = table.slice(0, 2)
+      issues = await validateCapFormat(headerTable, fileType, file)
```

Comment:
This function doesn't have a third parameter.

```suggestion
      issues = await validateCapFormat(headerTable, fileType)
```